test/mason/publish-tests/publishHelp.chpl

```diff
@@ -0,0 +1,18 @@
+
+use MasonPublish;
+use MasonUtils;
+
+proc test(arg: [] string) {
+  masonPublish(arg);
+}
+
+proc check () {
```

Comment:
Lets change this test so that it is checking the actual help output instead of the help output from the two arguments. Lets put the help output in the ``.good`` and compare against that. otherwise both outputs could be wrong in the same way.